JsonSchema.OpenApi/DiscriminatorKeyword.cs

```diff
@@ -55,7 +55,7 @@ public DiscriminatorKeyword(string propertyName, IReadOnlyDictionary<string, str
 		Mapping = mapping;
 		Extensions = extensions;
 
-		_json = JsonSerializer.SerializeToNode(this, OpenApiSerializerContext.OptionsManager.SerializerOptions);
+		_json = JsonSerializer.SerializeToNode(this, OpenApiSerializerContext.Default.JsonNode);
```

Comment:
Why is this JsonNode and not DiscriminatorKeyword? This seems like a mismatch

Reply:
oh... ignore that.  Just look at the logic stuff.  That's the only thing guaranteed to work so far.